src/packages/@ncigdc/components/Modals/CheckBoxModal.js

```diff
@@ -0,0 +1,83 @@
+// @flow
```

Comment:
CheckBoxModal.**tsx** ? :nerd_face:

Reply:
@lepsalex yes. They all should be ts/tsx.